types/react-native/index.d.ts

```diff
@@ -162,6 +162,31 @@ interface EmitterSubscription extends EventSubscription {
     remove(): void;
 }
 
+interface DEPRECATED_RCTExport<T> {
+	getConstants(): Function;
+}
+
+interface TurboModule extends DEPRECATED_RCTExport<void> {}
+
+interface Spec extends TurboModule {
+    addListener(eventName: string): void;
+    removeListeners(count: number): void;
+}
+
+declare class TurboModuleRegistry {
+	get<TurboModule>(name: string): (legacyModule: any) => TurboModule
+}
+
+interface NativeTVNavigationEventEmitter {
+	addListener(eventName: string, callback: Function): NativeEventEmitter;
+}
```

Comment:
Where is this export defined?

Reply:
As there is no export defined for it specifically, no need for this interface it seems.